PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/networking/modelcontrollers/Requester.java

```diff
@@ -88,6 +89,10 @@ public void destroy() {
         }
         networkTask = null;
         adResponseCallBack = null;
+        if (fetchAdIdInfoTask != null) {
+            fetchAdIdInfoTask.cancel(true);
+        }
+        fetchAdIdInfoTask = null;
```

Comment:
⛏️ That's probably very unlikely to happen but it would be safer first cancel `fetchAdIdInfoTask` and then clear `adResponseCallBack`. Otherwise both can race and still produce the same crash.